app/diagnostics.go

```diff
@@ -670,13 +670,17 @@ func (a *App) trackConfig() {
 		"enable_antivirus":              pluginActivated(cfg.PluginSettings.PluginStates, "antivirus"),
 		"enable_autolink":               pluginActivated(cfg.PluginSettings.PluginStates, "mattermost-autolink"),
 		"enable_aws_sns":                pluginActivated(cfg.PluginSettings.PluginStates, "com.mattermost.aws-sns"),
+		"enable_confluence":             pluginActivated(cfg.PluginSettings.PluginStates, "mattermost-plugin-com.mattermost.confluence"),
```

Comment:
Why did we missed that? :confused:

Reply:
Actually, the plugin `id` is `com.mattermost.confluence`: https://github.com/mattermost/mattermost-plugin-confluence/blob/4e3fd8a5a5fbedb71d929707ae66b0f16e08bf0c/plugin.json#L2

Reply:
Interesting.  I'm not sure why the downloadables are different than the ID.. https://user-images.githubusercontent.com/7575921/84047363-cb45c500-a970-11ea-8ede-1f4d575752d7.png

Reply:
Because the build tools are vastly different then https://github.com/mattermost/mattermost-plugin-starter-template. See also: https://github.com/mattermost/mattermost-plugin-confluence/issues/54